logstash-mixin-aws.gemspec

```diff
@@ -19,7 +19,7 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency "logstash-core-plugin-api", ">= 1.60", "<= 2.99"
   s.add_runtime_dependency 'logstash-codec-plain'
   s.add_runtime_dependency 'aws-sdk-v1', '>= 1.61.0'
-  s.add_runtime_dependency 'aws-sdk', '~> 2'
+  s.add_runtime_dependency 'aws-sdk', '~> 3'
```

Comment:
I you are afraid of the major version switch, then enable us with optimistic versioning for aws-sdk gem by using 
s.add_runtime_dependency 'aws-sdk', '>= 2'

Reply:
but this works with logstash 7.16.2 and aws-sdk 3